ee/docker-compose.ch.yml

```diff
@@ -79,7 +79,9 @@ services:
             - '8000:8000'
             - '8234:8234'
     plugins:
-        image: posthog/plugin-server:latest
+        build:
+            context: ../plugins
+            dockerfile: Dockerfile
```

Comment:
Alternatively I think this can be ran using the base PostHog image, just using `command: ./bin/plugin-server`